### DISCUSSION TITLE
Where has go2rtc streams page gone with the new Frigate layout?

### DISCUSSION THREAD

**Discussion by Cr4z33** [blakeblackshear/frigate#13677] [Ask A Question] [answered]
I see go2rtc yaml loaded and succesfully running in Frigate logs, but since when I have the new layout I don't know where the (go2rtc)  Streams / Add / Config" page has gone.

Also I can't reach the same page if using port 1984, but maybe that requires to run go2rtc externally separated from the built-in Frigate's version?

**Comment by banthungprong** [comment, marked as answer]
https://docs.frigate.video/guides/configuring_go2rtc/
`Navigate to port 1984 in your browser to access go2rtc's web interface.
If using Frigate through Home Assistant, enable the web interface at port 1984.
If using Docker, forward port 1984 before accessing the web interface.`

**Reply by Cr4z33** [reply]
Thanks mate although I never had to go and enable that port.

How do I do it? By opening it in my router or?

**Reply by NickM-27** [reply]
You enable it in the docker config

**Reply by Cr4z33** [reply]
> You enable it in the docker config

Sorry forgot to mention that I am running Frigate in Home Assistant so where should I go? 😅

**Reply by NickM-27** [reply]
To the extra config tab of addon settings

**Comment by Cr4z33** [comment]
I am confused as I don't think I got it.

Did you mean here?
![Screenshot_20240911_164737_Home Assistant](https://github.com/user-attachments/assets/2c6d5c4b-dffa-4e81-bea1-53bfb1cdc356)

**Reply by NickM-27** [reply]
yes, show disabled ports

**Reply by Cr4z33** [reply]
Thank you mate it's working now!
